microcosm_pubsub/message.py

```diff
@@ -17,14 +17,16 @@ def __init__(self,
                  message_id,
                  receipt_handle,
                  topic_arn=None,
-                 approximate_receive_count=None):
+                 approximate_receive_count=None,
+                 handler=None):
         self.consumer = consumer
         self.content = content
         self.media_type = media_type
         self.message_id = message_id
         self.receipt_handle = receipt_handle
         self.topic_arn = topic_arn
         self.approximate_receive_count = approximate_receive_count
+        self.handler = handler
```

Comment:
I am temporarily saving the "original" handler on the message. I don't intend to keep this.